AY2122S2-CS2103T-T13-3/tp
# Improve UG based on feedback

## Related issues
Resolves #126 

## Description
- Improved the UG based on feedback from our tutor

## How to test
1. cd to `docs/` folder
2. Run `bundle exec jekyll serve`
3. Open the server link (should be something like [http://127.0.0.1:4000](http://127.0.0.1:4000))

Alternatively, you may view the markdown file directly [here](https://github.com/AY2122S2-CS2103T-T13-3/tp/blob/da9e3aad890614c1cefe3eaaaa8a2e394c7ae4b1/docs/UserGuide.md)
